src/analytics/index.ts

```diff
@@ -1,9 +1,17 @@
 import { Identify, identify, init, track } from '@amplitude/analytics-browser'
-import { isProductionEnv } from '../utils/env'
 
 import { ApplicationTransport, OriginApplication } from './ApplicationTransport'
 
+export type Options = {
+  // URL of the proxy server
+  proxyUrl: string | undefined
+  isProductionEnv: boolean | undefined
+  commitHash: string | undefined
+}
+
 let isInitialized = false
+let isProductionEnv = false
+export let commitHash
```

Comment:
exporting a single variable without context here feels a bit odd. Could we store the entire options/config and export that instead? Also gives it clear typing too. We may not need all members yet but I can imagine future uses of these types of fields outside of initialization beyond just the commitHash.

Reply:
Love it